packages/contracts/contracts/SmartInvoicePayNowV2.sol

```diff
@@ -0,0 +1,62 @@
+//SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/security/Pausable.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+// error ValueMustNoBeZero();
+
+contract SmartInvoicePayNowV2 is Pausable, Ownable {
```

Comment:
There is no SmartInvoicePayNowV1, again not a fan of filename version control.

Reply:
i think it got buried in my initial pr comment, but this contract is just a placeholder I was using for testing for adding implementations and can be disregarded for now.